src/value_generators/src/DocumentGenerator.cpp

```diff
@@ -736,6 +739,25 @@ class RandomDateGenerator : public Generator<bsoncxx::types::b_date> {
     const std::chrono::milliseconds _max;
 };
 
+
+class IncGenerator : public Generator<int64_t> {
+public:
+    IncGenerator(const Node& node, GeneratorArgs generatorArgs)
+        : _step{node["step"].maybe<int64_t>().value_or(1)} 
+        {_counter = node["start"].maybe<int64_t>().value_or(1) + generatorArgs.actorId * node["multiplier"].maybe<int64_t>().value_or(1);}
```

Comment:
please ensure you run `clang-format` prior to merging

```sh
$ brew install clang-format
$ clang-format --version
#> clang-format version 11.0.0 # must be 11.0.0

clang-format -i ./src/**/*.cpp ./src/**/*.hpp
```

Reply:
Ran. Absolutely no output. Can I assume that everything is fine?